src/test/regress/sql/multi_alter_table_add_constraints.sql

```diff
@@ -417,11 +418,11 @@ SELECT "Constraint", "Definition" FROM table_checks WHERE relid='products'::regc
 
 \c - - - :worker_1_port
 
-SELECT "Constraint", "Definition" FROM table_checks WHERE relid='public.products_1450034'::regclass;
+SELECT "Constraint", "Definition" FROM table_checks WHERE relid='public.products_1450202'::regclass;
 
 \c - - - :master_port
-SET citus.next_shard_id TO 1450038;
-SET citus.next_placement_id TO 1450038;
+SET citus.next_shard_id TO 1450234;
```

Comment:
I know they were already there, but why we have them in the first place ? can we drop it altogether ?